.github/workflows/pr-build.yml

```diff
@@ -67,6 +67,20 @@ jobs:
         with:
           arguments: build integrationTests --stacktrace -PenableCoverage=true -PlocalDocker=true
 
+      - name: Get current version
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
if `matrix.is != ubuntu-latest`, then line 84 has `build with gradle` step for that. in that case, would be building the `java agent image` as well?

Reply:
no. because we will only publish this image for linux. We do not intend to publish this image for other operating systems (I'm not sure if that is even possible).